locale/en.po

```diff
@@ -911,59 +911,88 @@ msgstr "Starting"
 msgid "AUTO-EVO_POPULATION_CHANGED"
 msgstr "{0} population changed by {1} because of: {2}"
 
-#: ../src/auto-evo/RunResults.cs:458
+#: ../src/auto-evo/RunResults.cs:559
 msgid "POPULATION"
 msgstr "population:"
 
-#: ../src/auto-evo/RunResults.cs:467
+#: ../src/auto-evo/RunResults.cs:568
 msgid "WENT_EXTINCT_IN"
 msgstr "went extinct in {0}"
 
-#: ../src/auto-evo/RunResults.cs:473
+#: ../src/auto-evo/RunResults.cs:574
 msgid "PREVIOUS"
 msgstr "previous:"
 
-#: ../src/auto-evo/RunResults.cs:489
+#: ../src/auto-evo/RunResults.cs:590
 msgid "RUN_RESULT_SPLIT_FROM"
 msgstr "split off from {0}"
 
-#: ../src/auto-evo/RunResults.cs:502
+#: ../src/auto-evo/RunResults.cs:603
 msgid "RUN_RESULT_NICHE_FILL"
 msgstr "emerged to fill a niche"
 
-#: ../src/auto-evo/RunResults.cs:505
+#: ../src/auto-evo/RunResults.cs:604
+msgid "TIMELINE_NICHE_FILL"
+msgstr "[u]{0}[/u] has split into [u]{1}[/u] to fill a niche"
+
+#: ../src/auto-evo/RunResults.cs:609
 msgid "RUN_RESULT_SELECTION_PRESSURE_SPLIT"
 msgstr "emerged due to differing selection pressures"
 
-#: ../src/auto-evo/RunResults.cs:522
+#: ../src/auto-evo/RunResults.cs:610
+msgid "TIMELINE_SELECTION_PRESSURE_SPLIT"
+msgstr ""
+"[u]{0}[/u] has split into [u]{1}[/u] due to differing selection pressures"
+
+#: ../src/auto-evo/RunResults.cs:629
 msgid "RUN_RESULT_SPLIT_OFF_TO"
 msgstr "population in some patches split off to form new species {0}:"
 
-#: ../src/auto-evo/RunResults.cs:538
+#: ../src/auto-evo/RunResults.cs:645
 msgid "RUN_RESULT_HAS_A_MUTATION"
 msgstr "has a mutation"
 
-#: ../src/auto-evo/RunResults.cs:543
+#: ../src/auto-evo/RunResults.cs:650
 msgid "RUN_RESULT_GENE_CODE"
 msgstr "gene code:"
 
-#: ../src/auto-evo/RunResults.cs:554
+#: ../src/auto-evo/RunResults.cs:661
 msgid "RUN_RESULT_SPREAD_TO_PATCHES"
 msgstr "spread to patches:"
 
-#: ../src/auto-evo/RunResults.cs:562
+#: ../src/auto-evo/RunResults.cs:673
 msgid "RUN_RESULT_BY_SENDING_POPULATION"
 msgstr "{0} by sending: {1} population from patch: {2}"
 
-#: ../src/auto-evo/RunResults.cs:581
+#: ../src/auto-evo/RunResults.cs:692
+msgid "TIMELINE_SPECIES_MIGRATED_TO"
+msgstr "[u]{0}[/u] has migrated to {1}"
+
+#: ../src/auto-evo/RunResults.cs:699
+msgid "TIMELINE_SPECIES_MIGRATED_TO_MULTIPLE"
+msgstr "[u]{0}[/u] has migrated to multiple patches"
+
+#: ../src/auto-evo/RunResults.cs:706
 msgid "RUN_RESULT_POP_IN_PATCHES"
 msgstr "population in patches:"
 
-#: ../src/auto-evo/RunResults.cs:688
+#: ../src/auto-evo/RunResults.cs:816
 msgid "WENT_EXTINCT_FROM_PLANET"
 msgstr "went extinct from the planet"
 
-#: ../src/auto-evo/simulation/food_source/ChunkFoodSource.cs:64
+#: ../src/auto-evo/RunResults.cs:819
+msgid "TIMELINE_SPECIES_EXTINCT"
+msgstr "[u]{0}[/u] has gone extinct!"
+
+#: ../src/auto-evo/RunResults.cs:830
+msgid "TIMELINE_SPECIES_POPULATION_INCREASE"
+msgstr "[u]{0}[/u] populations have increased to {1}"
```

Comment:
I think using "population" without the plural sounds better here. As populations has the implication that they are like separate, multiple different populations.